modules/eco-setting-node-to-maintenance-mode-with-cr.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-setting-node-to-maintenance-mode-with-cr_{context}"]
+= Setting a node to maintenance mode with a NodeMaintenance custom resource
+
+You can put a node into maintenance mode with a `NodeMaintenance` custom resource (CR). When you apply a `NodeMaintenance` CR, all allowed pods are evicted and the node is shut down. Evicted pods are queued to be moved to another node in the cluster.
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in to the cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the following node maintenance CR, and save the file as `nodemaintenance-cr.yaml`:
++
+[source,yaml]
+----
+apiVersion: nodemaintenance.medik8s.io/v1beta1
+kind: NodeMaintenance
+metadata:
+  name: maintenance-example  <1>
+spec:
+  nodeName: node-1.example.com <2>
+  reason: "Node maintenance" <3>
+----
+<1> The name of the node maintenance CR
+<2> The name of the node to be put into maintenance mode
+<3> A plain text description of the reason for maintenance
++
+.  Apply the node maintenance schedule by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f nodemaintenance-cr.yaml
+----
+
+. Check the progress of the maintenance task by running the following command, replacing `<node-name>` with the name of your node:
```

Comment:
`<node-name>` -> nodeName

Since nodeName is indeed the node which we put under maintenance